src/midi/midi_mt32.cpp

```diff
@@ -626,6 +630,15 @@ bool MidiHandler_mt32::Open([[maybe_unused]] const char *conf)
 		mixer_channel->SetLowPassFilter(FilterState::Off);
 	}
 
+	// Double the baseline PCM prebuffer because MIDI is demanding
+	const auto render_ahead_ms = MIXER_GetPreBufferMs() * 2;
+
+	static constexpr auto max_frames_per_ms = 48;
```

Comment:
Are these sample frames? That's my best guess (48000 / 1000 = 48). If that's the case, could you please rename all occurrences of the word "frame" in this file to "sample_frame"? I find this particularly confusing because the word "frame" can mean [very different things](https://en.wikipedia.org/wiki/MIDI_timecode) when it comes to MIDI.

Reply:
Also, shouldn't this value be dependent on the mixer sample rate (frame rate) then?

Reply:
renamed all :+1:

Reply:
Switched to polling the mixer's actual audio frame rate (instead of using a maximum bound, here).